src/main/scala/gitbucket/core/util/JGitUtil.scala

```diff
@@ -1244,6 +1252,25 @@ object JGitUtil {
     git.log.add(startCommit).addPath(path).setMaxCount(1).call.iterator.next
   }
 
+  def getBranchesNoMergeInfo(git: Git, defaultBranch: String): Seq[BranchInfoSimple] = {
```

Comment:
Can we add a test case for this method? The same test case as `getBranches` would be enough.
https://github.com/gitbucket/gitbucket/blob/9caa73a5854ca4c5c32a27a1677cc39859428425/src/test/scala/gitbucket/core/util/JGitUtilSpec.scala#L147-L174